build.gradle

```diff
@@ -49,7 +49,7 @@ intellij {
     // Note: The IntelliJ version below needs to match the go plugin version as defined here:
     // https://plugins.jetbrains.com/plugin/9568-go/versions
     version = IJ_VERSION
-    type = 'CE'
+    type = 'IU'
```

Comment:
For some reason `CE` had been breaking the unit tests, causing `ClassNotFound` errors.